src/Platform/Microsoft.Testing.Platform/CommandLine/PlatformCommandLineProvider.cs

```diff
@@ -187,19 +163,19 @@ public Task<ValidationResult> ValidateCommandLineOptionsAsync(ICommandLineOption
 
         if (commandLineOptions.IsOptionSet(ExitOnProcessExitOptionKey))
         {
-            commandLineOptions.TryGetOptionArgumentList(ExitOnProcessExitOptionKey, out string[]? pid);
+            _ = commandLineOptions.TryGetOptionArgumentList(ExitOnProcessExitOptionKey, out string[]? pid);
             ApplicationStateGuard.Ensure(pid is not null);
             RoslynDebug.Assert(pid.Length == 1);
             int parentProcessPid = int.Parse(pid[0], CultureInfo.InvariantCulture);
             try
             {
                 // We let the api to do the validity check before to go down the subscription path.
                 // If we don't fail here but we fail below means that the parent process is not there anymore and we can take it as exited.
-                Process.GetProcessById(parentProcessPid);
+                _ = Process.GetProcessById(parentProcessPid);
             }
             catch (Exception ex)
```

Comment:
not your code, but we can be more specific and catch ArgumentException.
```suggestion
            catch (ArgumentException ex)
```